serde-plain-derive/src/lib.rs

```diff
@@ -0,0 +1,83 @@
+use proc_macro::TokenStream;
+
+fn get_ident(input: TokenStream) -> String {
+    let input = input.to_string();
+    
+    let mut x = 9;
+    let mut start_i = input
+        .find(" pub enum");
```

Comment:
Doesn't this completely fall apart if a doc comment is used with `///` that contains `pub enum`? I'm also confused of why it looks for SPACE + `pub enum` and not just `pub enum`.

Reply:
It probably would, but I added the space to possibly avoid some cases where it wouldn't work?

Reply:
I'm not sure when it comes across `" pub enum"`. When does that happen? (other than an indented enum).